lib/trie/child_storage.go

```diff
@@ -12,6 +12,8 @@ import (
 // ChildStorageKeyPrefix is the prefix for all child storage keys
 var ChildStorageKeyPrefix = []byte(":child_storage:default:")
 
+const ErrChildTrieDoesNotExist = "child trie does not exist at key"
```

Comment:
Change that to an exported global error

```suggestion
var ErrChildTrieDoesNotExist = errors.New("child trie does not exist")
```